docs/configuration.md

```diff
@@ -285,13 +285,13 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
-  <td><code>spark.files.userClassPathFirst</code></td>
+  <td><code>spark.driver.userClassPathFirst, spark.executor.userClassPathFirst</code></td>
```

Comment:
It would kinda suck to duplicate the actual description, since they'd be identical.